### PR TITLE
feat: add plugins to menu, refactor menus

### PR DESCRIPTION
**Pull request by Nybkox**

### CODE REVIEW

**Comment by Nybkox** [comment]
@leviszaboo, could you fill the options menu?

**Comment by leviszaboo** [comment]
Sure, but maybe not today.

File: menu_items/plugins.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+CURRENT_FILE="${BASH_SOURCE[0]}"
+CURRENT_DIR="$(dirname -- "$(readlink -f -- "$0")")"
+ROOT_DIR="$(dirname "$CURRENT_DIR")"
```
Comment (leviszaboo) [inline menu_items/plugins.sh:5]: this produces an error on my machine and is fixed if i set it to ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)"

Comment (leviszaboo) [reply menu_items/plugins.sh:5]: by this, I mean the ROOT_DIR

File: menu_items/options.sh [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+  local option=$1
+  local name=""
+  for i in "${!options[@]}"; do
+    if [[ "${options[$i]}" == "$option" ]]; then
+      name="${titles[$i]}"
+      break
+    fi
+  done
+
+  if [[ $(read_option_from_state "$option") == true ]]; then
```
Comment (leviszaboo) [inline menu_items/options.sh:23]: this condition is buggy if the global option is set in .tmux.conf

Comment (leviszaboo) [inline menu_items/options.sh:23]: this condition is buggy if the global option is set in .tmux.conf

File: scripts/state.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ read_option_from_state() {
     touch "$STATE_FILE"
   fi
 
-  local row=$(grep "^$1" "$STATE_FILE")
+  local row=$(cat "$STATE_FILE" | grep "$1")
```
Comment (leviszaboo) [inline scripts/state.sh:11]: i don't think this changes anything, but the other one did not work with options for some reason

**Comment by leviszaboo** [comment]
I added a couple of toggle options, I don't think it breaks anything. I also added some comments about issues that I had, but I'm not gonna have time to work on them in the following days.

**Pull request merged**